app/src/main/java/de/westnordost/streetcomplete/quests/general_fee/AddGeneralFee.kt

```diff
@@ -11,7 +11,8 @@ class AddGeneralFee : OsmFilterQuestType<Boolean>() {
 
     override val elementFilter = """
         nodes, ways, relations with
-         (tourism = museum or leisure = beach_resort or tourism = gallery)
+         (tourism = museum or leisure = beach_resort or tourism = gallery
+         or office=coworking or amenity=coworking_space)
```

Comment:
Formatting. Please always add spaces at the operators and use the parenthesis and indentation like elsewhere in the filters.

On a separate note, I think a coworking space has no place here. All coworking spaces I know are member-only. I.e. comparable to a fitness studio - you pay a monthly fee and then may use it. Maybe there are exceptions. But maybe there are also exceptiosn for fitness studios and fitness studios are also not in this list because most, as written, are membership-based. (Hence, impossible to answer for surveyor)

Reply:
Hmm, in Poland there are several that are existing to make nearby objects more interesting - for example in malls or operated by a public entity such as library (though I would need to check has it survived COVID)

OSM Wiki mentions that they are typically paid, but still recommended fee tag so I thought that such free ones are occuring also elsewhere ( https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcoworking_space#Additional_tags )

Reply:
Some libraries have rooms where you can work/read quietly, but those are something else than coworking spaces.

Reply:
This specific one was different, but seems relatively unique. Removed coworking spaces here, modified wiki pages to make clear that vast majority is paid.